roles/openshift_version/tasks/set_version_containerized.yml

```diff
@@ -1,7 +1,4 @@
 ---
-- set_fact:
-    l_use_crio: "{{ openshift_docker_use_crio | default(false) }}"
```

Comment:
This right here is what I call a 'bad practice'

Reply:
Could be, but in fairness it is what was previously considered the way to use local facts in this code base.